packages/backend-data/src/convert_js_resolvers.ts

```diff
@@ -71,12 +78,15 @@ export const convertJsResolverDefinition = (
 
     const resolverName = `Resolver_${resolver.typeName}_${resolver.fieldName}`;
 
+    const amplifyEnvironmentName =
+      scope.node.tryGetContext('amplifyEnvironmentName') ?? 'NONE';
```

Comment:
This context var `amplifyEnvironmentName` looks like is set by data construct exclusively? Is this the `branchName` for CI/CD deployments and `NONE` for sandbox?

Reply:
Yes, this is this the branchName for CI/CD deployments and NONE for sandbox.

Reply:
than we are using the wrong naming here. `amplifyEnvironmentName` suggests amplify wide name for the environment which is not the case here. May I suggest `amplifyApiEnvironmentName`?

Reply:
The value is already used in the data construct. To use the new name without a breaking change we would need to set both variables, then it might be confusing why there are two variables with the same value. I think it is a better option to continue using the existing one.

Reply:
I'm not asking to change in the data construct but rather the new variables created in the `backend-data` to which this value is assigned. As well as the new variable set in the `convert_js_resolvers.ts`. It's not a branch name since it's applicable for sandboxes as well.

Reply:
Oh sorry, you mean `const amplifyEnvironmentName` not what is context?

Reply:
yes, I understand context vars are harder to change, but we should not create more vars with this naming (such as `amplifyBranchName`)